src/process_manager.h

```diff
@@ -181,6 +181,7 @@ typedef struct thread_s {
   int m_orig_block_id; /**< original block id from a trace*/
   int m_orig_thread_id; /**< adjusted block id */
   int m_block_id; /**< block id */
+  uint64_t m_trace_id;
```

Comment:
Is this trace_id or thread_id?

Reply:
I deleted `m_trace_id`, it was never used